pkg/events/v1alpha1/events.go

```diff
@@ -33,16 +33,24 @@ const (
 	GovernorApplicationLinkRequestsEventSubject = "applinks.requests"
 	// GovernorApplicationTypesEventSubject is the subject name for application type events (minus the subject prefix)
 	GovernorApplicationTypesEventSubject = "applicationtypes"
+	// GovernorNotificationTypesEventSubject is the subject name for notification type events (minus the subject prefix)
+	GovernorNotificationTypesEventSubject = "notificationtypes"
+	// GovernorNotificationTargetsEventSubject is the subject name for notification target events (minus the subject prefix)
+	GovernorNotificationTargetsEventSubject = "notificationtarget"
+	// GovernorNotificationPreferencesEventSubject is the subject name for notification preference events (minus the subject prefix)
+	GovernorNotificationPreferencesEventSubject = "notificationpreference"
```

Comment:
maybe these should be `notifications.foo`, then an addon could subscribe to all events about notifications if it wanted

Reply:
would it be a good idea to also use this for `notification-preferences`, e.g. `users.notificationpreferences` or `users.preferences.notifications`?

Reply:
should this be handled by an update event on user instead? what limitations might that impose on downstream systems?

Reply:
arguably none. I implemented this as an effort of following what everything else had done, but I can hardly come up with a reason for the downstream systems to catch a user notification update event, unless we are trying to cache the notification preferences downstream.

It could make sense if we are to support a more sophisticated user preferences, e.g. to support more than just notification preferences, then it might be good to have a `users.preferences.*`

Reply:
yeah I would argue we can implement it at that time then?